File: maoy-spider/src/entity/City.ts
import {Entity, Column, PrimaryGeneratedColumn } from "typeorm";
@Entity()
export class City {
    @PrimaryGeneratedColumn()
    id: number;

    @Column({type:'varchar'})
    cityId: string;

    @Column({type:'varchar'})
    pinyin: string;
    
    @Column({type:'varchar'})
    name: string;

    @Column({type:'varchar'})
    rank: string;
    
    @Column({type:'varchar'})
    acronym: string;
    
    @Column({type:'varchar'})
    chineseFullName: string;

    @Column({type:'double'})
    lat: number;

    @Column({type:'double'})
    lng: number;
    
    @Column({type:'varchar'})
    hot: string;
}

File: maoyan-server/src/main/java/com/xs/maoyan/model/City.java
package com.xs.maoyan.model;

public class City {
	private Integer id;
	private String cityId;
	private String pinyin;
	private String name;
	private String rank;
	private String acronym;
	private String chineseFullName;
	private Double lat;
	private Double lng;
	private String hot;
	
	public Integer getId() {
		return id;
	}
	public void setId(Integer id) {
		this.id = id;
	}
	public String getCityId() {
		return cityId;
	}
	public void setCityId(String cityId) {
		this.cityId = cityId;
	}
	public String getPinyin() {
		return pinyin;
	}
	public void setPinyin(String pinyin) {
		this.pinyin = pinyin;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getRank() {
		return rank;
	}
	public void setRank(String rank) {
		this.rank = rank;
	}
	public String getAcronym() {
		return acronym;
	}
	public void setAcronym(String acronym) {
		this.acronym = acronym;
	}
	public String getChineseFullName() {
		return chineseFullName;
	}
	public void setChineseFullName(String chineseFullName) {
		this.chineseFullName = chineseFullName;
	}
	public Double getLat() {
		return lat;
	}
	public void setLat(Double lat) {
		this.lat = lat;
	}
	public Double getLng() {
		return lng;
	}
	public void setLng(Double lng) {
		this.lng = lng;
	}
	public String getHot() {
		return hot;
	}
	public void setHot(String hot) {
		this.hot = hot;
	}
}
